pkg/mock_httpserver_test.go

```diff
@@ -49,31 +49,38 @@ func StartMockHttpServer() {
 			case PrefixConfigResponse.STATUS_400:
 				response.StatusCode400().SetErrors([]string{"A 400 error has occurred"})
 				w.WriteHeader(http.StatusBadRequest)
-				w.Write([]byte(response.StatusCode400().ToJson()))
+				resp400, _ := response.StatusCode400().ToJson()
```

Comment:
resp400, err := response.StatusCode400().ToJson()
add error check also and if there is an error server shall return 400 status object ji

Reply:
updated as discussed